bookmark.lua

```diff
@@ -32,76 +32,110 @@ jumpdrive.write_to_book = function(pos, sender)
 
 end
 
+local function hasNil(tPos)
+	if nil == tPos
+		or nil == tPos.x
+		or nil == tPos.y
+		or nil == tPos.z
+	then
+		return true
+	end
+	return false
+end
+
+local function sanitizeAndSetCoordinates(meta, tPos)
```

Comment:
Also here should maybe follow most used coding style mostly used elsewhere in code.

Reply:
you are absolutely correct, need to clean that up.

Reply:
also correct that there is no hurry for this.